ArduPlane/Parameters.h

```diff
@@ -240,7 +240,7 @@ class Parameters {
         // Battery monitoring parameters
         //
         k_param_battery = 166,
-        k_param_rssi_pin,               // unused, replaced by rssi_ library parameters - 167
+        k_param_volz_protocol,               // unused, replaced by rssi_ library parameters - 167
```

Comment:
sorry, this is not allowed. You need to use a new slow. Check the new g2 param list for more space.

Reply:
Ok, followed the "Adding a New Parameter" will read about the g2 param list, thanks